codemods/custom-property-replacer/variable-mapping.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
How about we scope those changes into "version folders", so we make them "immutable"?

```
codemods/custom-property-replacer/v10/variable-mapping.json
```

Then in the future we can have more of them...